### ISSUE TITLE
Errors When Building Project

### ISSUE THREAD

**Issue by samdow** [EmergentOrder/onnx-scala#1]
When I clone the project and try to build it using `sbt +publishLocal` as in the README, I get the following errors:
```[error] (programGeneratorJVM / update) sbt.librarymanagement.ResolveException: download failed: org.bytedeco.javacpp-presets#onnx;1.3.0-1.4.4-SNAPSHOT!onnx.jar
[error] (coreJS / Compile / compileIncremental) java.lang.StackOverflowError
[error] (coreJVM / Compile / compileIncremental) java.lang.StackOverflowError
[error] (coreNative / Compile / compileIncremental) java.lang.StackOverflowError
[error] (coreDottyJVM / update) sbt.librarymanagement.ResolveException: unresolved dependency: org.emergentorder.onnx#onnx-scala-common_0.10;1.3.0-0.1.0-SNAPSHOT: not found
```

Any suggestions on how to fix this would be appreciated!

**Comment by EmergentOrder** [comment]
Actually, I didn't delete my ivy cache before testing just now, will do that to see if I get the dependency error.

**Comment by EmergentOrder** [comment]
Ok, I can reproduce the dependency error now. It looks you need to do the default build first, and then the cross-build, in order to not get the error. I'll fix this so you can invoke the cross-build on its own.

**Comment by samdow** [comment]
Thanks for the quick reply and for building the project! I had some issues with the dependency with the default build because I think only the snapshot for JavaCPP 1.4.3 is released (it worked if I downgraded the sbt to use 1.4.3 instead of 1.4.4). I ended up downloading 1.4.4 and building from source, figuring that you've been contributing so much to it, it's probably necessary. Figured I would just note this if someone else runs into this issue while building. Looking forward to playing around with the project! Thanks for your help!
